doc/design/linc_permissions.md

```diff
@@ -1,43 +1,50 @@
-• As a LINC Archive user, I shall be able to render neuroglancer layers view similar to DANDI
+### Use case 1
+Use the `External Services` button in the `FileBrowser` to load a Neuroglancer viewer with
+a single layer of the corresponding asset.  The resulting URL can be shared between LINC team members.  This use case extends the capabilities in the DANDI Archive to view assets in a private S3 bucket.
 
-#### Relevant Links
+### Use case 2
+Set a single Neuroglancer viewer programmatically to visualize multiple assets (e.g. dMRI, HiP-CT) as layers.
 
-https://github.com/orgs/lincbrain/projects/2?pane=issue&itemId=54650571
+### Use case 3
+Use case 2 + Load the tractography-generated streamlines (i.e. `trk` file) into memory,
+convert the streamlines into the Neuroglancer skeleton format, and visualize in the Neuroglancer
+viewer as an additional layer.
 
-https://github.com/neuroscales/ngtools/blob/main/notebooks/show_tract.ipynb
+## Requirements
 
-https://github.com/google/neuroglancer/issues/507
+1. Zarr archives are stored in a private AWS S3 bucket
+2. Provide direct access to the LINC users of the assets on the private S3 bucket using a URI.
 
 ## Implementation Options
 
-#### Issue read-only credentials for S3 bucket. Quick set up.
+### Issue read-only credentials for S3 bucket. Quick set up.
 
 Does not resolve rendering issue. Would resolve accessing the asset in Jupyter Notebook, but would require flaky Python code.
 
-#### Integrate read-only credentials for S3 bucket into LINC Archive, LINC CLI
+### Integrate read-only credentials for S3 bucket into LINC Archive, LINC CLI
 
-###### CloudFront distribution with Origin Access Identity control for relevant S3 buckets
+#### CloudFront distribution with Origin Access Identity control for relevant S3 buckets
 
 - S3 Asset Bucket is served via a CloudFront distribution
-- User hits a get_presigned_cookie/ API endpoint in LINC Archive. In the endpoint response is a valid cookie that allows the user's browser to be able to access the asset via their browser
-
-Rendering via Neuroglancer
-
-- (Might need to modify fork of neuroglancer to handle CloudFront asset path in same format that it handles S3 asset path)
-
-Accessing via LINC Archive API
+- User navigates to LINC Archive (i.e. `lincbrain.org`) to authenticate.  Thereby the user hits a get_presigned_cookie/ API endpoint. In the endpoint response is a valid cookie that allows the user's browser to be able to access the asset via their browser.
 
+Render with hosted Neuroglancer
+- Host Neuroglancer fork on S3 bucket with static site hosting enabled. Create an AWS CloudFront distribution to sit as a verified proxy for permission to many S3 assets at once via presigned cookies.  Provide CNAME for CloudFront Distribution URL to `neuroglancer.lincbrain.org` so that cookies generated by LINC Archive (also living at a `*.lincbrain.org`) can be shared between the API and the Neuroglancer site.
```

Comment:
```suggestion
- Host Neuroglancer fork on S3 bucket with static site hosting enabled. Create an AWS CloudFront distribution to sit as a verified proxy with permission to access many S3 assets at once via presigned cookies.  Provide CNAME for CloudFront Distribution URL to `neuroglancer.lincbrain.org` so that cookies generated by LINC Archive (also living at a `*.lincbrain.org`) can be shared between the API and the Neuroglancer site.
```